docs/resource/host.md

```diff
@@ -31,6 +31,20 @@
 | `x86` | 32-bit x86 |
 <!-- endsemconv -->
 
+**type:** `host.cpu`
+
+<!-- semconv host.cpu -->
+| Attribute  | Type | Description  | Examples  | Requirement Level |
+|---|---|---|---|---|
+| `host.cpu.vendor_id` | string | Processor manufacturer identifier. A maximum 12-character string. | `GenuineIntel` | Opt-In |
+| `host.cpu.family` | int | Numeric value specifying the family or generation of the CPU. | `6` | Opt-In |
+| `host.cpu.model` | int | Model identifier. It provides more granular information about the CPU, distinguishing it from other CPUs within the same family. | `6` | Opt-In |
+| `host.cpu.model_name` | string | Model designation of the processor. | `11th Gen Intel(R) Core(TM) i7-1185G7 @ 3.00GHz` | Opt-In |
+| `host.cpu.stepping` | int | Stepping or core revisions. | `1` | Opt-In |
+| `host.cpu.frequency` | double | Clock frequency in MHz. | `3000.0` | Opt-In |
+| `host.cpu.cache_size` | int | The amount of level 2 memory cache available to the processor (in Bytes). | `12288000` | Opt-In |
```

Comment:
Based on #99 I think these would probably make more sense as a metric (at least if these can change). @frzifus any thoughts about this?

Reply:
From my understanding the cache size even in a vm highly depends on the hardware, right? Means the only way to change the cache size would be to swap the CPU. As long as those attribute values are associated with a specific cpu, that should not matter.

Thats different for the CPU frequency. On a normal linux machine it can be changed by writting a number to `/sys/..`. (see [scaling_setspeed](https://superuser.com/questions/533507/how-can-i-manually-set-the-cpu-frequency)) and the other way around for example aws instances [offer a boosted frequency](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/processor_state_control.html).

Reply:
Yes to my findings this one (coming from `/proc/cpuinfo`) refers to the L2 cache of the CPU, which is fixed. 
For reference LLC or L3 cache can be allocated dynamically in some systems but we don't refer to this one with this attribute.

For the frequency it's weird to me that the value is changing since I would expect it to be fixed to the model's value and not the actual freq of the cores'. However, I did some tests and it's actually changing so it's seems that I'm wrong here.

Reply:
Frequency will be added as metric.